src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -82,6 +93,15 @@ function CloseAccountPage(props) {
         return errors;
     };
 
+    const getErrorMessage = () => {
+        let error = '';
+        const hasSharedPolicies = PolicyUtils.hasSharedPolicies(props.policies, props.allPolicyMembers);
+        if (hasSharedPolicies) {
+            error = props.translate('closeAccountPage.hasSharedPolicies');
+        }
+        return error;
```

Comment:
Should we early return here?
```suggestion
        if (!hasSharedPolicies) {
            return '';
        }
        return props.translate('closeAccountPage.hasSharedPolicies');;
```